app/attributes/attributeformmodel.h

```diff
@@ -54,6 +54,7 @@ class  AttributeFormModel : public QAbstractListModel
       RememberValue, //!< Remember value (whether to remember the value)
       Field, //!< Field
       FieldIndex, //!< Index
+      FieldState, //!< State of the field, Valid, InvalidInput,..
```

Comment:
it is more about field value than field itself... hmm.. maybe ValueValidity? CurrentValueValidity? not sure